### DISCUSSION TITLE
Projects section here for news and announcements

### DISCUSSION THREAD

**Discussion by castrojo** [cncf/toc#1294] [Ideas]
Hey ya'll!

Over the past few weeks the projects team @ the CNCF has been working with TAG Contributor Strategy to shore up content here: https://contribute.cncf.io/resources/project-services/

The problem we're trying to solve is that it's difficult for project maintainers to keep track of all the resources available to them, and things are constantly being added/changed, etc. 

We do have a newsletter that we send to maintainers, but how many people are actually reading that? 🦖 So I'd like to propose a section here called Projects, which will include:

- Every newsletter we send projects to also be posted here. This will also serve as an archive and is much easier to maintain than PRing the newsletter to the website
- (Awesome part): This allows maintainers to post and ask questions about the newsletters if they need details.
- They can also give us feedback on contribute.c.i that we can use to directly improve the site. 

I've been tagged in Slack a few times already since Kubecon and I literally have canned replies that I am using over and over to respond to individual emails and PMs. So the idea is to centralize the questions that projects have that aren't appropriate for a service ticket. For example it would have been awesome to have a running feedback thread for the Project Pavilion so we can start to gather information right away instead of "remember to respond to the survey!". 

And the async nature means that we can also start sharing information as a group instead of a bunch of slack DMs and emails. Examples for other things:

- More "know before you go" style reminders for projects leading up to events.
- "New landscape features landed, here are the new fields being added and some examples" 
- "Does anyone have a really good ADOPTERS.md I can derive from?"
- We can use QR codes for signage at events that can direct people here to ask questions. 

And I'd also like to recommend that we do it in this repo for a few reasons: 
- Centralization, the TOC and TAGs could pop in and see what things maintainers are asking about, and the upvoting feature let's us quickly gauge what people are interested in the most. 
- It would help socialize the services provided and would also expose the top level discussions to project maintainers, since many of them affect their projects anyway. 

Thoughts?

**Comment by jkowall** [comment]
I was not aware of this, awesome stuff. I am now joined https://contribute.cncf.io/about/contributor-growth/ on Slack

**Comment by TheFoxAtWork** [comment]
SGTM thank yoU!

**Comment by castrojo** [comment]
Still working on this, the newsletter is HTML and I'm investigating a way to convert it in an automated way.
